code/__DEFINES/dcs/signals.dm

```diff
@@ -90,6 +90,7 @@
 #define COMSIG_ATOM_BUMPED "atom_bumped"
 ///from base of atom/ex_act(): (severity, target)
 #define COMSIG_ATOM_EX_ACT "atom_ex_act"
+	#define COMPONENT_BLOCK_EX_ACT (1<<0)
```

Comment:
SPACING

Reply:
this is how all signal return flags are setup 😳